1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,41 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+
+  const p = document.createElement("p");
+
+  const selectColor = document.createElement("select");
+
```

Comment:
The tests don't quite pass - it could be completed by adding a disabled option at the beginning using something like the following:

```suggestion
 firstOption = document.createElement("option");
  firstOption.disabled = true;
  firstOption.setAttribute("selected", true);
  firstOption.setAttribute("hidden", true);
  firstOption.value = "Please choose a colour";
  selectColor.appendChild(firstOption);
```

I don't really think this was asked for in the exercise README though, and it's pretty horribly contrived, so I'm perfectly happy with what you've done here! :)